### ISSUE TITLE
Bring Doc and Press info off forum & onsite

### ISSUE THREAD

**Issue by iheartcryptocoin** [super3/peercoin_website_old#284]
Linking to forum posts for the doc and press section is not professional.  Additionally the Press kit info need to be completely redone.  

Additionally JBT has moved on so we need to update the contact info for the point person to make an interview request with Sunny

**Comment by super3** [comment]
Agreed. We need a singular page or section for the press that is on the website not the forum.

**Comment by iheartcryptocoin** [comment]
We also need to establish standards for what news articles will be listed, this initial pull is based off of Sentinelv's forum post.  However there are many other articles we may wish to include.  It might also be good to create a "cryptos" in the news section below the Peercoin news section

**Comment by super3** [comment]
I think we should have 3 sections. Major media, crypto new articles, and casual mentions in media.

**Comment by iheartcryptocoin** [comment]
Additional To Do

Add #peercoin live feed to news page.  C'kay suggests Rebel Mouse 
Am currently exploring free service. https://www.rebelmouse.com/static/welcome/publishers-and-media.html

Other suggestions?

**Comment by iheartcryptocoin** [comment]
For Major News will search these top listed sites 
-http://www.ebizmba.com/articles/news-websites

-http://www.comscoredatamine.com/2012/12/most-read-online-newspapers-in-the-world-mail-online-new-york-times-and-the-guardian/

and collect 

A) Peercoin Mention
B) Bitcoin/Crypto generally mentioned

We also need to collect all of the past Weekly Newsletters and add therm as toggles in a Community News tab on the News and Events page

Currently am creating a CSV with all of the convention info and all of the news articles, then we simply process and output required HTML.  Add new events and articles and reprocess as needed.

https://onedrive.live.com/redir?resid=1198B965207FB78!238&authkey=!ALvBCfbiUpcwAMs&ithint=file%2c.xlsx
